packages/bpk-component-checkbox/src/BpkCheckbox.scss

```diff
@@ -103,36 +103,31 @@
   }
 
   &__checkmark {
-    &::before {    
+
+    &::before, &::after {
       position: absolute;
+      border-radius: $bpk-border-size-lg;
+      transform: rotate(45deg);
+      content: '';
+      background-color: $bpk-color-white;
+    }
```

Comment:
Using combination of before and after to abstract things common to both pseudoclasses.